config/initializers/carrierwave.rb

```diff
@@ -1,11 +1,17 @@
+require "pathname"
 require "whitehall/asset_manager_storage"
 require "carrier_wave/sanitized_file"
 
 CarrierWave.configure do |config|
   config.storage_engines[:asset_manager] = "Whitehall::AssetManagerStorage"
   config.storage Whitehall::AssetManagerStorage
   config.enable_processing = false if Rails.env.test?
-  config.cache_dir = Rails.root.join "carrierwave-tmp"
+  uploads_root = if !ENV.fetch("GOVUK_UPLOADS_ROOT", "").empty?
```

Comment:
Nit: 

Not 💯 off the top of my head whether the Rails helper methods are available in the initialisers - but you should be able to simplify this to:
```suggestion
  uploads_root = if ENV["GOVUK_UPLOADS_ROOT"].present?
```
Avoids having to think through a double negative.

Reply:
Thanks, that reads much better!